### DISCUSSION TITLE
Module for Nuxt 3

### DISCUSSION THREAD

**Discussion by oyedejioyewole** [phosphor-icons/homepage#287] [Show and tell]
Hey guys 👋, I spent the entire day working on a `Nuxt 3` module to port the `@phosphor-icons/vue` project.

I hope you guys like it 😅
<https://github.com/OyewoleOyedeji/nuxt-phosphor-icons>

**Comment by rektdeckard** [comment]
This is cool, thanks @OyewoleOyedeji! As a Vue (and Nuxt) noob, can you explain what this does that our Vue lib doesn't? Thanks!

If you'd like to have this resource added to our [Community Projects](https://github.com/phosphor-icons/homepage#community-projects) list, please make a PR over on that repo!

**Reply by oyedejioyewole** [reply]
Well I made a module which use an extension in Nuxt 3. It allows you take advantage of auto imports to use `@phosphor-icons/vue` components without explicitly writing an import statement inside each component.

I hope this helps 😁

**Comment by insanity54** [comment]
Thank you! I am confused though. Beyond installation, the readme doesn't have a basic usage example so I'm not sure how to use the icons. Is there like a `<PhosphorIcon>` component, or does it use `<i>` tag or something like that?

**Reply by oyedejioyewole** [reply]
Hey, thanks for trying out the module. The `README` generally describes the module, you should visit <https://nuxt-phosphor-icons.vercel.app> for the documentation.
